### PR TITLE
cpu to support bitwise ops

### PR DESCRIPTION
**Pull request by liqunfu**
### Description
BitwiseAnd, BitwiseNot, BitwiseOr, and BitwiseXor are added in ONNX 1.13. Add cpu kernels to support these ops.

### Motivation and Context
this is for ORT 1.14 release

### CODE REVIEW

**Comment by edgchen1** [comment]
should we add unit tests for the new ops?

File: onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc [hunk 1]
```diff
@@ -118,7 +118,6 @@
         "^test_div_uint8_cuda",
         "^test_add_uint8_cuda",
         "^test_roialign_aligned_*",
-        "^test_bitwise_*",
```
Comment (skottmckay) [inline onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc:121]: Please update \csharp\test\Microsoft.ML.OnnxRuntime.Tests.NetCoreApp\InferenceTest.netcore.cs as well to remove the exclusions.

File: onnxruntime/core/providers/cpu/math/element_wise_ops.cc [hunk 2]
```diff
@@ -1155,7 +1191,122 @@ Status BitShift<T>::Compute(OpKernelContext* context) const {
 }
 
 template <typename T>
-class Sin final : public OpKernel {
+Status BitwiseAnd<T>::Compute(OpKernelContext* context) const {
```
Comment (gramalingam) [inline onnxruntime/core/providers/cpu/math/element_wise_ops.cc:1194]: It looks like a lot of boilerplate code that should be common to all binary elementwise ops. Just curious why there is no common template for this.

**Review by gramalingam — APPROVED** [review]
LGTM, modulo Scott's suggestion. I think it should be possible to use a common template for all binary elementwise ops (and one for unary elementwise ops) to avoid redundant code.

**Comment by liqunfu** [comment]
> LGTM, modulo Scott's suggestion. I think it should be possible to use a common template for all binary elementwise ops (and one for unary elementwise ops) to avoid redundant code.

I agree that elementwise code can be templated so that common code not be replicated. However it shall be a code restructure task by itself instead done with this PR. I also often find templated macros hard to read and impossible to debug so we shall be careful to avoid using micros for the task.

**Review by gramalingam — APPROVED** [review]

**Pull request merged**